example/config/managedk8s.yaml

```
@@ -4,6 +4,8 @@ providers:                   # contains information about known providers
   metadata:
     foo: bar
   args:
+    # opsPodLabels: # pod labels that will be added to diki ops pods
```

I think that we should rename this so that it is clear that these labels are added to an existing set of labels.

```suggestion
    # additionalOpsPodLabels: # pod labels that will be added to diki ops pods
```